include/tdycore.h

```diff
@@ -43,12 +45,12 @@ typedef enum {
 PETSC_EXTERN const char *const TDyMPFAOGmatrixMethods[];
 
 typedef enum {
-  MPFAO_DIRICHLET_BC=0,  /* Dirichlet boundary condiiton */
-  MPFAO_NEUMANN_BC,       /* Neumann zero-flux boundary condition */
-  MPFAO_SEEPAGE_BC       /* Seepage boundary condition */
-} TDyMPFAOBoundaryConditionType;
+  DIRICHLET_BC=0,  /* Dirichlet boundary condiiton */
```

Comment:
This is fine, but we're probably going to replace it with the new boundary condition logic.